aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -886,13 +1543,8 @@ def register_resolver(func: Callable):
                 else:
                     self._static_routes.append(_route)
 
-                route_key = item + rule
-                if route_key in self._route_keys:
-                    warnings.warn(
-                        f"A route like this was already registered. method: '{item}' rule: '{rule}'",
-                        stacklevel=2,
-                    )
-                self._route_keys.append(route_key)
+                self._create_route_key(item, rule)
```

Comment:
Had to do this to keep Cyclomatic Complexity happy

Reply:
It's an improvement. Great catch.